types/kendo-ui/index.d.ts

```diff
@@ -10112,10 +10140,42 @@ declare namespace kendo.dataviz.ui {
         width?: number|Function;
     }
 
+    interface ChartSeriesItemMarkersFromBorder {
+        color?: string|Function;
+        width?: number|Function;
+    }
+
+    interface ChartSeriesItemMarkersFrom {
+        background?: string|Function;
+        border?: ChartSeriesItemMarkersFromBorder;
+        size?: number|Function;
```

Comment:
On this and other uses of the type `Function`, please consider using a more specific type as discussed here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes